src/pybind/mgr/volumes/fs/subvolume.py

```diff
@@ -189,6 +189,29 @@ def get_subvolume_path(self, spec):
             raise VolumeException(-e.args[0], e.args[1])
         return path
 
+    def get_dir_names(self, path):
```

Comment:
minor nit: maybe call it `get_dir_entries`?

Reply:
get_dir_entries may suggest getting names of not just directories, but other file types.

Reply:
I'm leaning towards making this routine generic (if time permits) -- caller could request for matching file types to be filtered.